react/features/local-recording/recording/WavAdapter.js

```diff
@@ -40,40 +40,28 @@ export class WavAdapter extends RecordingAdapter {
         }
 
         const p = new Promise((resolve, reject) => {
-            navigator.getUserMedia(
-
-                // constraints - only audio needed for this app
-                {
-                    audioBitsPerSecond: WAV_SAMPLE_RATE * WAV_BITS_PER_SAMPLE,
-                    audio: true,
-                    mimeType: 'application/ogg' // useless?
-                },
-
-                // Success callback
-                stream => {
-                    this._audioContext = new AudioContext();
-                    this._audioSource
-                     = this._audioContext.createMediaStreamSource(stream);
-                    this._audioProcessingNode
-                      = this._audioContext.createScriptProcessor(4096, 1, 1);
-                    this._audioProcessingNode.onaudioprocess = e => {
-                        const channelLeft = e.inputBuffer.getChannelData(0);
-
-                        // https://developer.mozilla.org/en-US/docs/
-                        // Web/API/AudioBuffer/getChannelData
-                        // the returned value is an Float32Array
-                        this._saveWavPCM(channelLeft);
-                    };
-                    this._isInitialized = true;
-                    resolve();
-                },
-
-                // Error callback
-                err => {
-                    logger.error(`Error calling getUserMedia(): ${err}`);
-                    reject();
-                }
-            );
+            this._getAudioStream(0)
```

Comment:
This `0` will need to be replaced by state['features/base/settings'].micDeviceID.